sql/core/src/test/resources/sql-tests/results/datetime.sql.out

```diff
@@ -0,0 +1,10 @@
+-- Automatically generated by org.apache.spark.sql.SQLQueryTestSuite
```

Comment:
It might be better to remove the package name so we don't need to change all the generated files when we move this class.